test/Sema/accessibility_typealias.swift

```diff
@@ -27,7 +27,7 @@ public final class ReplayableGenerator<S: Sequence> : IteratorProtocol {
 }
 
 struct Generic<T> {
-  fileprivate typealias Dependent = T
+  fileprivate typealias Dependent = T // expected-note 2{{type declared here}}
```

Comment:
Fixed one of the walks somewhere?

Reply:
Yeah this is 26bd877fda81a310b3278a0cd58410dd1d2137f0, the `bool` return of `walkToTypeReprPost` was used incorrectly, which was uncovered by this patch :)

Reply:
Very nice :). I'm surprised that was the only one to be honest!